$NetBSD: patch-configure.in,v 1.1 2015/09/07 18:57:47 gdt Exp $

In pkgsrc, we don't provide Qt as a framework.

It's not clear how/if this should be addressed upstream.

--- configure.in.orig	2015-01-02 23:02:55.000000000 +0000
+++ configure.in
@@ -214,20 +214,9 @@ case "$target" in
       mac/libusb/usb.o "
       USB_LIBS="-framework  IOKit -framework CoreFoundation"
       AC_DEFINE(HAVE_LIBUSB, 1)
- 
-      # On Mac, use frameworks for includes and library files.
-      # Macports uses a non-standard build of Qt with an added 
-      # QT_INSTALL_FRAMEWORKS flag, so look for thier custom
-      # version first.  That will fail on 'normal' Qt so fall 
-      # back to QT_INSTALL_LIBS.
-      QT_FW_OR_LIBS=$($QMAKE -query QT_INSTALL_FRAMEWORKS)
-      if test $QT_FW_OR_LIBS = "**Unknown**" ; then
-         QT_FW_OR_LIBS=$($QMAKE -query QT_INSTALL_LIBS)
-      fi
-      QT_LIBS="-F$QT_FW_OR_LIBS -framework QtCore"
-      QT_INC="$QT_FW_OR_LIBS"
-      QT_INC_OPT="-F"
-      QT_SYSINC_OPT="-iframework"
+
+
+      # In pkgsrc, we use the installed qt, not the framework.
       ;;
     *-*-freebsd*)
 	GBSER=gbser_posix.o
